subsys/tracing/sysview/tracing_sysview.h

```diff
@@ -29,9 +30,21 @@ void sys_trace_mutex_unlock(struct k_mutex *mutex);
 
 static inline void sys_trace_thread_info(struct k_thread *thread)
 {
+#if IS_ENABLED(CONFIG_THREAD_NAME)
```

Comment:
if you look at sysview.c, there is similar code doing exactly the same thing, time to put this in a helper function and use it in both places and avoid duplication.

Reply:
Fixed